### PR TITLE
Add documentation for phases endpoint.

### PR DESCRIPTION
**Pull request by funkaoshi**

### CODE REVIEW

File: source/includes/phases.md [hunk 1]
```diff
@@ -1 +1,70 @@
...
+Parameter | Description
+----------|-------------
+name      | Returns the phase resource with specified name
+slug      | Returns the phase resource with the specified slug
+
+
+## Get a Specific Phase
+
+```http
+GET /api/v2/phase/1/ HTTP/1.1
```
Comment (lmontopo) [inline source/includes/phases.md]: I think the url should be /api/v2/phases/1/  ... you're missing an 's'.

File: source/includes/phases.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+    "description": "Application security requirements, generally used to assist requirements analysts.",
+    "name": "Requirements",
+    "slug": "requirements",
+    "tip": "One-time tasks that you can verify. Similar to traditional functional requirements or user stories."
+}
+```
+
+This endpoint retrieves a single phase resource, as specified by the id parameter.
+
+**`GET /api/v2/applications/{phase_id}/`**
```
Comment (lmontopo) [inline source/includes/phases.md]: should this be `/api/v2/phases/{phase_id}/` instead of 'api/v2/applications/{phase_id}/'?

**Comment by funkaoshi** [comment]
Will force push all the typos cleaned up.

File: source/includes/phases.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+
+This endpoint retrieves a single phase resource, as specified by the id parameter.
+
+**`GET /api/v2/phases/{phase_id}/`**
+
+### URL Parameters
+
+Parameter      | Description
+-------------- | ---------------
+application_id | The id of the application to retrieve
```
Comment (lmontopo) [inline source/includes/phases.md]: {phase_id} instead of {application_id}, and then in the description the word 'application' should be changed to 'phase'

**Comment by funkaoshi** [comment]
Should now be typo free!

**Comment by lmontopo** [comment]
looks good to me! :+1:

File: source/includes/phases.md [hunk 4]
```diff
@@ -1 +1,70 @@
...
+
+This endpoint retrieves a single phase resource, as specified by the id parameter.
+
+**`GET /api/v2/phases/{phase_id}/`**
+
+### URL Parameters
+
+Parameter  | Description
+---------- | ---------------
+phase_id   | The id of the phase to retrieve
```
Comment (geoffwhittington) [inline source/includes/phases.md]: `phase_id` isn't identified as a field in a response. Are we missing this field in the response? Do we mean to call it `id`  like we do for other artifacts?

Comment (funkaoshi) [inline source/includes/phases.md]: Yeah, I mentioned this in LCB. you look up details for a phase with a field that we don't send back.

Comment (geoffwhittington) [inline source/includes/phases.md]: I'm not on my game tonight :)

**Pull request merged**